src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx

```diff
@@ -142,6 +142,101 @@ To host synthetics job managers, your system must meet the minimum requirements
     </Callout>
   </Collapser>
 
+  <Collapser
+    id="podman-requirements"
+    title={<><img src="/images/synthetic_logo_podman.webp" title="Podman icon" alt="Podman icon" style={{ height: '35px', width: '40px' }}/>Podman container system environment requirements</>}
+  >
+    <table>
+      <thead>
+      <tr>
+        <th style={{ width: "165px" }}>
+          Compatibility for
+        </th>
+
+        <th>
+          Requirements
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          Operating system
+        </td>
+
+        <td>
+          <DNT>**Linux kernel:**</DNT> 3.10 or higher<br/>
```

Comment:
are we confident on these requirements?

Reply:
Yes, those are the same requirements as that of the Docker.